buildkite/graphql/cluster.graphql

```diff
@@ -0,0 +1,70 @@
+fragment ClusterFields on Cluster {
+    id
+    uuid
+    name
+    description
+    emoji
+    color
+}
+
+query getCluster($orgSlug: ID!, $id: ID!) {
+    organization(slug: $orgSlug) {
+        cluster(id: $id) {
+            ...ClusterFields
+        }
+    }
+}
+
+# @genqlient(for: "ClusterCreateInput.description", pointer: true)
+# @genqlient(for: "ClusterCreateInput.emoji", pointer: true)
+# @genqlient(for: "ClusterCreateInput.color", pointer: true)
```

Comment:
I thought you only had to do it like this if you are accepting a struct type as input for the mutation? Otherwise you could add these annotations above the individual arguments below on lines 24-26

Reply:
You do, I don't personally like the inline style however can change it if there are strong opinions on it. I prefer directives at the top bet not a hill I'll die on.

Reply:
Neither, as long as it works 👍